codes/kotlin/chapter_stack_and_queue/linkedlist_stack.kt

```diff
@@ -0,0 +1,86 @@
+/**
+ * File: linkedlist_stack.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd (1023632660@qq.com)
+ */
+
+package chapter_stack_and_queue
+
+class LinkedListStack(
```

Comment:
class header comment

(These code had been merged. Please fix them)

Reply:
ok, I will fix this after this chapter has been merged.